pkgs/development/libraries/gstreamer/rs/default.nix

```diff
@@ -0,0 +1,244 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, fetchpatch
+, writeText
+, rustPlatform
+, meson
+, ninja
+, python3
+, pkg-config
+, rust
+, rustc
+, cargo
+, cargo-c
+, nasm
+, gstreamer
+, gst-plugins-base
+, gst-plugins-bad
+, gtk4
+, cairo
+, csound
+, dav1d
+, libsodium
+, libwebp
+, openssl
+, pango
+, Security
+, gst-plugins-good
+, nix-update-script
+# TODO: required for case-insensitivity hack below
+, yq
+, moreutils
+# specify a limited set of plugins to build if not all supported plugins
+, plugins ? null
+}:
+
+let
+  # populated from meson_options.txt
```

Comment:
Did you use a command to generate this?

Reply:
No, I did not. I can clarify that in the comment